frontend/src/design/components/layout/DefaultSidebar.js

```diff
@@ -82,7 +82,8 @@ export const DefaultSidebar = ({ openDrawer, onOpenDrawerChange }) => {
     const organizationsSection = {
       title: 'Organizations',
       path: '/console/organizations',
-      icon: <BiIcons.BiBuildings size={15} />
+      icon: <BiIcons.BiBuildings size={15} />,
+      active: config.modules.organizations.active
```

Comment:
As we discussed we don't want to disable organizations. So we can remove this line :)

Reply:
Yes, I will update this.

Reply:
Updated with [9bb870c](https://github.com/awslabs/aws-dataall/pull/612/commits/9bb870c8763d34903d3e1901fc6f62c1bc69db65)